substrate/utils/wasm-builder/src/wasm_project.rs

```diff
@@ -841,9 +850,7 @@ fn build_bloaty_blob(
 				"-C target-cpu=mvp -C target-feature=-sign-ext -C link-arg=--export-table ",
 			);
 		},
-		RuntimeTarget::Riscv => {
-			rustflags.push_str("-C target-feature=+lui-addi-fusion -C relocation-model=pie -C link-arg=--emit-relocs -C link-arg=--unique ");
-		},
+		RuntimeTarget::Riscv => (),
```

Comment:
Can we move the target features to be an inherent function on `RuntimeTarget`?

Reply:
Same as earlier. It's local to `wasm_project.rs`. Let's reconsider if this ever changes.

Reply:
Your comment about build-std made sense, given that it has 2x sites, and improves validation in the other site. I'd set this as a common sense threshold what to merge to `RustTarget`.